tests/app/test_accessibility_statement.py

```
@@ -8,13 +8,21 @@ def test_last_review_date():
 
     # test local changes against main for a full diff of what will be merged
     statement_diff = subprocess.run(
-        [f"git diff --unified=0 --exit-code origin/main -- {statement_file_path}"], stdout=subprocess.PIPE, shell=True
+        [f"git diff --unified=0 --exit-code origin/main -- {statement_file_path}"],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        shell=True,
     )
 
     date_format = "%d %B %Y"
 
     # if statement has changed, test the review date was part of those changes
     if statement_diff.returncode == 1:
+        assert not statement_diff.stderr, (
```

Should this come up a few lines, ie up out of the if statement for checking the returncode?

I'm guessing we want to fail this test everytime there is stderr, rather than just if the return code is 1 and there is stderr?

---

good point (part of the reason we were surprised is because we'd expected the statuscode on error to be, eg, 127 (given we pass in the `--statuscode` flag to git diff to make it return 0 on no diff, 1 on diff)